src/definitions.ts

```diff
@@ -2,18 +2,18 @@ import { Breadcrumb, Response } from '@sentry/types';
 
 import { CapacitorOptions } from './options';
 
-declare module '@capacitor/core' {
-  interface PluginRegistry {
-    SentryCapacitor: SentryCapacitorPlugin;
-  }
-}
+// declare module '@capacitor/core' {
```

Comment:
This will get merged?

Reply:
Left it there because I was not sure if we were going to fully drop v2 or not, turns out we don't need it regardless because we're not exposing the native bridge to the users. This just types the native bridge plugin.